### DISCUSSION TITLE
Benchmarks page

### DISCUSSION THREAD

**Discussion by lifeisfoo** [dragonflydb/dragonfly#2741] [General]
"A 25X increase in throughput compared to Redis" is an impressive result but I would like to know more about it.

Dragonfly's documentation still doesn't have a page on ["How to Benchmark it"](https://github.com/dragonflydb/dragonfly/discussions/1688#discussioncomment-6688261) and no specific information are provided about versions used and the setup process involved.

I think such page would be useful to all the ones who like to evaluate Dragonfly on their own before migrating.
Could you publish a detailed guide on how to compare them? And also update the claimed benchmarks adding a timestamp of the run and the software versions used?

**Comment by gaby** [comment]
Also worth comparing against Garnet, their benchmarks show Redis and Dragonfly pretty close.

https://microsoft.github.io/garnet/docs/benchmarking/results-resp-bench

Thoughts? @romange

**Comment by romange** [comment]
You are right, we have not added any proper documents about benchmarking yet. I just raised the following PR 
https://github.com/dragonflydb/documentation/pull/219/files

will continue working on it in the coming days but it already has enough info for evaluating Dragonfly performance.
